lib/trino-parquet/src/main/java/io/trino/parquet/reader/Int64TimestampMillisColumnReader.java

```diff
@@ -44,6 +44,7 @@ protected void readValue(BlockBuilder blockBuilder, Type type)
             type.writeLong(blockBuilder, packDateTimeWithZone(epochMillis, UTC_KEY));
         }
         else if (type instanceof TimestampType) {
+            // The existing int64-millis-timestamp support is just converting timestamp to microsecond precision.
```

Comment:
If this comment is meant to explain what the next line does, it seems redundant.

Reply:
it is, just put some comments in relevant places to highlight key timestamp handling steps; 
such comments are in a separate commit that is not intended to reach beyond draft pr.